utils/config.js

```diff
@@ -11,7 +13,7 @@ if (typeof window === "undefined") {
     POST_LOGOUT_REDIRECT_URI: process.env.POST_LOGOUT_REDIRECT_URI,
     SESSION_COOKIE_SECRET: process.env.SESSION_COOKIE_SECRET,
     SESSION_COOKIE_LIFETIME: process.env.SESSION_COOKIE_LIFETIME,
-    MONGODB_URI: process.env.MONGODB_URI,
+    MONGODB_URI: mongodb_uri.mongodb_uri(),
```

Comment:
I don't believe this line needs to change (actually, it shouldn't even work if you change it). The environment variables exposed to the application code are defined in the `next.config.js` file.